salt/salt.spec

```diff
@@ -638,12 +638,15 @@ Requires(pre):  %fillup_prereq
 Salt ssh is a master running without zmq.
 it enables the management of minions over a ssh connection.
 
-%package tests
+%package -n python3-salt-testsuite
 Summary:        Unit and integration tests for Salt
 Requires:       %{name} = %{version}-%{release}
+Requires:       python3-pytest >= 7.0.1
```

Comment:
Once we change to singlespec, we should make this one conditional (matching the python version)

Reply:
Do we have a plans to turn it to true singlespec?

Reply:
We don't have a plan yet, certainly not in the "Replace Salt Toaster" project. But it's something we might do after we evaluated using singlespec for Uyuni. That's something we want to take advantage of the new reduced Python stack.